bot/teamsBot.ts

```diff
@@ -160,43 +159,24 @@ export class TeamsBot extends TeamsActivityHandler {
           );
           break;
         }
-      }
+        case "view queue": {
+          try {
+            const teamMembers = await getNamesOfTeamMembers(context);
+            const queueMembers = this.activeQueue.getNamesInQueue(teamMembers);
 
-      if (txt.startsWith("view queue")) {
-        try {
-          const queueIdString: string = txt.replace("view queue", "").trim();
-          if (queueIdString === "") {
             if (this.activeQueue) {
-              await context.sendActivity(this.activeQueue.entriesToString());
+              await context.sendActivity(queueMembers);
             } else {
               await context.sendActivity(
```

Comment:
We probably want to show the questions here along with the member names. Let me know if you think that's out of the scope of this PR and I can work on that after this is merged.

Reply:
Changed this in latest commit